ajax/libs/tipograf/1.0.2/tipograf.css

```diff
@@ -0,0 +1,110 @@
+/**
+ * tipograf - Lightweight CSS typography base for your next project
+ * @version v1.0.1
```

Comment:
@x09326 we usually rely on the git tag and npm version, so just make sure we added the same as upstream, if so, this won't be a problem if the content contains the wrong version number.